examples/all-clusters-minimal-app/all-clusters-common/all-clusters-minimal-app.zap

```diff
@@ -1524,6 +1540,29 @@
               "maxInterval": 65344,
               "reportableChange": 0
             }
+          ],
+          "events": [
+            {
+              "name": "StartUp",
+              "code": 0,
+              "mfgCode": null,
+              "side": "server",
+              "included": 1
+            },
+            {
+              "name": "ShutDown",
```

Comment:
The ShutDown and Leave events are optional... why are they included?

Reply:
Correct! I see that in the spec they are marked as optional, however, in the XML file they are defined as mandatory:
`./src/app/zap-templates/zcl/data-model/chip/basic-information-cluster.xml`

Please take a look and let me know which one is correct. 

Note: As a followup to this PR (or maybe as part of this PR) I'll crosscheck Spec vs XML definitions.

Reply:
The XML is wrong here...  And yes, we need to align it with the spec.  Separate PR for that, please.